homeassistant/components/sensor/template.py

```diff
@@ -88,14 +89,25 @@ def __init__(self, hass, device_id, friendly_name, unit_of_measurement,
         self._state = None
         self._icon_template = icon_template
         self._icon = None
+        self._entities = entity_ids
+
+    @asyncio.coroutine
+    def async_added_to_hass(self):
+        """Register callbacks."""
+        state = yield from async_get_last_state(self.hass, self.entity_id)
+        if state:
+            self._state = state.state
 
         @callback
         def template_sensor_state_listener(entity, old_state, new_state):
             """Called when the target device changes state."""
-            hass.async_add_job(self.async_update_ha_state, True)
+            self.hass.async_add_job(self.async_update_ha_state, True)
 
         async_track_state_change(
```

Comment:
Should this not be added on HASS_START?

At this point, the entities it calculates from is not necessarily added